tool/tsh/db.go

```diff
@@ -252,13 +271,42 @@ func onDatabaseConfig(cf *CLIConf) error {
 	default:
 		return trace.BadParameter("unknown database protocol: %q", database)
 	}
-	switch cf.Format {
+
+	format := strings.ToLower(cf.Format)
```

Comment:
+1 for ignoring the case of the format. Do we have consistency with our other CLI tools in this regard? We may want to apply the same treatment e.g. for `tctl`, although preferably with a separate PR.